roles/contiv_network/tasks/main.yml

```diff
@@ -35,6 +35,9 @@
 - name: copy systemd units for netplugin
   copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
 
+- name: copy bash auto complete file for netctl
+  shell: sudo cp /usr/bin/contrib/completion/bash/netctl /etc/bash_completion.d/ && sudo rm -rf /usr/bin/contrib
```

Comment:
@mapuri the intention was not having any non-executables lying around in /usr/bin/ and so I explicitly do a `rm` as a part of the ansible task. The file contrib/completion/bash/netctl is sourced to the shell and should not be executed by the user directly. Was just being skeptic about getting into a scenario where we have another netctl (auto completion file needs to have the same name as the cli binary) lying in inside /usr/bin, even if in separate folder inside. If you think it should be ok, then I will make the change.